libcore/FileConflictDialog.vala

```diff
@@ -36,54 +36,36 @@ public class Files.FileConflictDialog : Granite.MessageDialog {
         NEWEST
     }
 
+    public Files.File source { get; construct; }
+    public Files.File destination { get; construct; }
+    public Files.File dest_dir { get; construct; }
```

Comment:
These should be `private get`, no?

Reply:
I thought construct properties had to be public get but maybe I'm wrong - I'll check.  There's probably a lot of other instances with this pattern in my code!